crates/client/rpc-core/src/utils.rs

```diff
@@ -47,6 +47,16 @@ pub(crate) fn _encode_base64(data: &[u8]) -> String {
     general_purpose::STANDARD.encode(data)
 }
 
+pub fn to_tx(request: BroadcastedTransaction) -> Result<Transaction> {
```

Comment:
Can't we implement `TryFrom` trait on `Transaction` to avoid this util function?